pkgs/applications/networking/instant-messengers/rambox-pro/default.nix

```diff
@@ -0,0 +1,46 @@
+{ autoPatchelfHook, electron, fetchurl, makeDesktopItem, makeWrapper, nodePackages, nss, stdenv, xdg_utils, xorg }:
+
+stdenv.mkDerivation rec {
+  pname = "rambox-pro-${version}";
+  version = "1.1.2";
+
+  dontBuild = true;
+  dontStrip = true;
+
+  buildInputs = [ nss xorg.libxkbfile ];
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper nodePackages.asar ];
+
+  src = fetchurl {
+    url = "https://github.com/ramboxapp/download/releases/download/v${version}/RamboxPro-${version}-linux-x64.tar.gz";
+    sha256 = "0rrfpl371hp278b02b9b6745ax29yrdfmxrmkxv6d158jzlv0dlr";
+  };
+
+  patches = [ ./remove-bash-args.patch ];
```

Comment:
I wonder if `patchShebangs` would be sufficient here or not...
We try to avoid hosting patches in-tree unless absolutely necessary.

Reply:
Possibly - the problem I'm having to address here is twofold:

1.) Removing extra junk from the shebang line *but* also adding it as a "set [mode]" later on. If patchShebangs can handle that then I'm more than happy to merge it.